app/assets/stylesheets/components/common/_news_signup_inpage.scss

```diff
@@ -0,0 +1,531 @@
+    }
+
+    @include respond-to($mq-l) {
+      margin-top: 10px;
+    }
+
+    @include respond-to($mq-l) {
+      margin-top: 0;
+    }
+
+    .news-signup-inpage__privacy-text {
+      @include body(13, 16);
+      margin-bottom: 0;
+
+      a::after {
+        margin-bottom: -4px;
+      }
+
+      span {
+        display: block;
+        @include respond-to($mq-s) {
+          display: inline;
+        }
+
+      }
+
+      @include respond-to($mq-s) {
+        position: absolute;
+        top: 50%;
+        left: 50%;
+        transform: translate(-50%, -50%);
+        white-space: nowrap;
+      }
+
+      @include respond-to($mq-3col-layout) {
+        @include body(11, 24);
+      }
+
+      @include respond-to($mq-l) {
+        @include body(13, 24);
+      }
+
+      .ie8 & {
+        position: relative;
+        top: 10px;
+        left: auto;
+      }
+
+      .ie7 & {
+        position: relative;
+        top: 10px;
+        left: auto;
+      }
+
+    }
+
+  }
+
+  &.news-signup-inpage--in-article .news-signup-inpage__image {
+    display: none;
+  }
+
+  .news-signup-inpage__container {
+    position: relative;
+  }
+
+  &.news-signup-inpage--in-article.news-signup-inpage--with-image .news-signup-inpage__image {
+    display: block;
+    background: transparent 50% 30% image_url('news_signup_test/bg.jpg') no-repeat;
+    background-size: cover;
+    height: 140px;
+
+    @include device-pixel-ratio() {
+      background-image: image_url('news_signup_test/bg@2x.jpg');
+    }
+  }
+
+  &.news-signup-inpage--in-article.news-signup-inpage--with-image .news-signup-inpage__image {
+    @include respond-to($mq-news-signup-inpage-article) {
+      height: auto;
+      position: absolute;
+      top: 0;
+      left: 0;
+      bottom: 0;
+      width: 33.333333333%;
+      background-position: 50% 50%;
+    }
+  }
+
+  &.news-signup-inpage--in-article {
+    display: block;
+    margin-top: $baseline-unit*4;
+    margin-bottom: $baseline-unit*2;
+  }
+
+  &.news-signup-inpage--in-article.news-signup-inpage--with-image .news-signup-inpage__container {
+    @include respond-to($mq-news-signup-inpage-article) {
+      margin-left: 33.333333333%;
+    }
+  }
+
+  &.news-signup-inpage--in-article .news-signup-inpage__container {
+    padding: 20px;
+  }
+
+  .news-signup-inpage__container__intro h3 {
+    @include body(22, 26);
+    color: $color-green-dark;
+    margin-top: 0;
+  }
+
+  .news-signup-inpage__container__intro p {
+    @include body(14, 14);
+  }
+
+  .news-signup-inpage__container__form input[type="email"] {
+    @include body(16, 16);
+    border: 1px solid $color-newsletter-input-border;
+    height: auto;
+    padding: $baseline-unit*2;
+    margin-right: $baseline-unit;
+    margin-bottom: $baseline-unit*2;
+
+    @include respond-to($mq-news-signup-inpage-article) {
+      width: 55%;
+      display: inline;
+      margin-bottom: $baseline-unit;
+    }
+  }
+
+  .news-signup-inpage__container__form .button {
```

Comment:
Same here, personally prefer wrapping all these statements in a single `.news-signup-inpage__container__form`, but this is just a personal preference and up to you